File: java-maven/plugins/kafka/src/test/java/{{ .PackagePath }}/{{ .NameWord }}/scenarios/KafkaScenario.java
package {{ .Package }}.{{ .NameWord }}.scenarios;

import io.gatling.javaapi.core.ScenarioBuilder;
import {{ .Package }}.{{ .NameWord }}.cases.KafkaActions;

import static io.gatling.javaapi.core.CoreDsl.scenario;

public final class KafkaScenario {

    private KafkaScenario() {
    }

    public static ScenarioBuilder create() {
        return scenario("Kafka Scenario")
                .exec(KafkaActions.sendMessage);
    }
}

File: kotlin-maven/plugins/kafka/src/test/kotlin/{{ .PackagePath }}/{{ .NameWord }}/scenarios/KafkaScenario.kt
package {{ .Package }}.{{ .NameWord }}.scenarios

import io.gatling.javaapi.core.CoreDsl.scenario
import io.gatling.javaapi.core.ScenarioBuilder
import {{ .Package }}.{{ .NameWord }}.cases.KafkaActions

object KafkaScenario {
    fun create(): ScenarioBuilder =
        scenario("Kafka Scenario")
            .exec(KafkaActions.sendMessage)
}
